rest-service/manager_rest/dsl_functions.py

```diff
@@ -49,6 +55,15 @@
             'path',
         ],
     },
+    'cloudify': {
+        'connection_parameters': [
+            'host',
+            'username',
+            'password',
+            'tenant',
+            'cert',
```

Comment:
let's not require `cert`, since we do allow http (not https). Also, even if we do use https, we can still omit the cert, and then we'll fall back to checking in the built-in CA store. That'll be the case when you're connecting to a manager that uses a cert signed by a public CA (eg. you used letsencrypt)